lib/internal/main/watch_mode.js

```diff
@@ -64,11 +66,23 @@ let exited;
 
 function start() {
   exited = false;
+
+  let kEnv = {};
+  for (const arg of process.execArgv) {
+    if (StringPrototypeStartsWith(arg, '--env-file')) {
```

Comment:
please use `getOptionValue` instead of iterating over all of `process.execArgv`

Reply:
In fact, I've already applied `getOptionValue`, but when I used `getOptionValue('--env-file')` with multiple --env-file options set, I had a problem that I could only get the last --env-file information.